### ISSUE TITLE
Unpin Tiles from Start Menu

### ISSUE THREAD

**Issue by astanix** [Sycnex/Windows10Debloater#119]
This button doesn't appear to do anything. I still have to manually unpin everything from the start menu after debloating and pushing unpin tiles.

**Comment by Gray-0men** [comment]
It takes a moment to work and it won't unpin things that are pinned yet not installed (some bloat on the  start is just an ad where when you click it, it'll install the app then open it)

**Comment by astanix** [comment]
It's leaving things that have the uninstall option. IMO debloat itself should be completely clearing out 
 all the games/apps/extras in the start menu when done.

After debloat:
![after debloat](https://user-images.githubusercontent.com/13404887/55990514-d2dae280-5c75-11e9-8eba-393e0972b56b.png)

10 min after clicking unpin:
![10 min after clicking unpin](https://user-images.githubusercontent.com/13404887/55990526-d79f9680-5c75-11e9-922d-2994dd6e9a77.png)

this can be 'uninstalled'
![uninstall](https://user-images.githubusercontent.com/13404887/55990531-da01f080-5c75-11e9-8f45-773db9e4f7f0.png)

**Comment by Gray-0men** [comment]
Was windows update running in the background doing updates? 
From the look at the background these screen shots seem to be going off a fresh install of Win10. 
And the network icon shows internet connection so if updates are running in the background then yeah some apps are going to get skipped because the computer can't uninstall (apps in this case) and install (updates in this case) at the same time. It's the same resource conflict that happens when you try to uninstall and install something and windows throws an error telling you to wait.

Also just to point this out.. the first screen shot it's not "after", I can see the debloat running right there with the light blue background with yellow text going over the dark blue default powershell background. You'll need to wait until the debloat is finished before clicking anything else, I haven't gotten the GUI to spawn multiple powershell windows per button press yet where each command runs in parallel (I don't think it will work either for the same conflict stated above).

One thing to note that's not in the description yet is that you **can't do multiple buttons at the same time**, they don't queue up and they don't run in parallel, meaning, if you were to look in your second screen shot where you say it's 10 mins after clicking unpin.. I can still see the debloat still running and happening for the same reason I stated before. That light blue background floating above the powershell window with yellow text going across is literally the debloat running so absolutely no button on the GUI will work until the debloat is complete. Including the unpin tiles button. 

In the 3rd screen shot I can STILL tell the debloat is running because the GUI is showing "not responding" which is exactly what it will do if you try to click another command before the previous one is NOT finished.. you can double check this yourself by running debloat all and trying to click on any other button.. any of them. They won't work and the GUI window will show not responding

**Comment by Mojacko** [comment]
Hi i can confirm this issue. tried it on fresh install of windows 10[1809] version 17763.1
even after restart the tiles are still there

**Comment by Gray-0men** [comment]
Was your fresh install connected to the internet and was it installing updates?

**Comment by Mojacko** [comment]
No. my computer was not connected to any network during that time.

**Comment by xsisbest** [comment]
From my experience using the sysprep and the stand alone debloaters, it only affects user accounts that have never been logged into. The only solution for those that are trying to debloat your existing profile is to maybe copy all your stuff to the administrator profile. Then remove your profile. Then run the debloat script and any new user that logs in will have it debloated. Unless it's changed over the past year that's how it worked for me in the beginning. I just use the sysprep version as I mainly want it ran during my MDT deployment in a task sequence.

**Comment by Sycnex** [comment]
This should now be resolved in the latest merged pull request.
